jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListView.java

```diff
@@ -0,0 +1,671 @@
+                                return "";
+                        }
+                    } else if ( row.getStatus().equals( "Completed" ) ) {
+                        return HumanTaskResources.INSTANCE.css().taskCompleted();
+                    }
+
+                }
+                return null;
+            }
+        };
+
+        extendedPagedTable.setEmptyTableCaption( constants.No_Tasks_Found() );
+
+        selectionModel = new NoSelectionModel<TaskSummary>();
+        selectionModel.addSelectionChangeHandler( new SelectionChangeEvent.Handler() {
+            @Override
+            public void onSelectionChange( SelectionChangeEvent event ) {
+                boolean close = false;
+                if ( selectedRow == -1 ) {
+                    selectedRow = extendedPagedTable.getKeyboardSelectedRow();
+                    extendedPagedTable.setRowStyles( selectedStyles );
+                    extendedPagedTable.redraw();
+
+                } else if ( extendedPagedTable.getKeyboardSelectedRow() != selectedRow ) {
+                    extendedPagedTable.setRowStyles( selectedStyles );
+                    selectedRow = extendedPagedTable.getKeyboardSelectedRow();
+                    extendedPagedTable.redraw();
+                } else {
+                    close = true;
+                }
+
+                selectedItem = selectionModel.getLastSelectedObject();
+
+                presenter.selectTask(selectedItem, close);
+            }
+        } );
+
+        noActionColumnManager = DefaultSelectionEventManager
+                .createCustomManager( new DefaultSelectionEventManager.EventTranslator<TaskSummary>() {
+
+                    @Override
+                    public boolean clearCurrentSelection( CellPreviewEvent<TaskSummary> event ) {
+                        return false;
+                    }
+
+                    @Override
+                    public DefaultSelectionEventManager.SelectAction translateSelectionEvent( CellPreviewEvent<TaskSummary> event ) {
+                        NativeEvent nativeEvent = event.getNativeEvent();
+                        if ( BrowserEvents.CLICK.equals( nativeEvent.getType() ) &&
+                            // Ignore if the event didn't occur in the correct column.
+                            extendedPagedTable.getColumnIndex( actionsColumn ) == event.getColumn() ) {
+                                return DefaultSelectionEventManager.SelectAction.IGNORE;
+                        }
+                        return DefaultSelectionEventManager.SelectAction.DEFAULT;
+                    }
+                } );
+        extendedPagedTable.setSelectionModel( selectionModel, noActionColumnManager );
+        extendedPagedTable.setRowStyles( selectedStyles );
+    }
+
+    @Override
+    public void initColumns( ExtendedPagedTable<TaskSummary>  extendedPagedTable ) {
+        initCellPreview( extendedPagedTable );
+        Column<TaskSummary, ?> taskIdColumn = initTaskIdColumn();
```

Comment:
You're right. Sorry I missed that, there were a lot of refactors since I first implemented this PR. I'll fix it